test/jdk/java/io/ObjectStreamClass/ObjectStreamClassCaching.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This update is incorrect, should be:

```suggestion
 * Copyright (c) 2021, 2022, Oracle and/or its affiliates. All rights reserved.
```

Reply:
Corrected in next commit.